srcpkgs/Dwarf-Therapist/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'Dwarf-Therapist'
+pkgname=Dwarf-Therapist
+version=41.2.1
+revision=1
+build_style=cmake
+makedepends="qt5-declarative-devel libcap-devel hicolor-icon-theme"
+depends="dwarffortress"
+short_desc="Management tool designed to run side-by-side with Dwarf Fortress"
+maintainer="Robert Stancil <robert.stancil@mavs.uta.edu>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/Dwarf-Therapist/Dwarf-Therapist/"
+distfiles="https://github.com/Dwarf-Therapist/Dwarf-Therapist/archive/v${version}.tar.gz"
+checksum=ec954daff6e03d9d44153ed3df90685de3a7a07431fd47639d83b2f5b3f62e4b
+
+post_install() {
+	vbin dist/ptrace_scope/patch_df_ptracer
+	vbin dist/ptrace_scope/ptrace_cap_wrapper
```

Comment:
Is this wrapper supposed to be suid / have capabilities? Afaik those won't be kept when installed with `vbin`

Reply:
I dont think they need any specific suid. To be honest, this mr has been created a long time ago.

But I'm pretty sure I used this package ootb